dotnet/src/webdriver/Internal/Logging/ILogContext.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace OpenQA.Selenium.Internal.Logging
+{
+    public interface ILogContext : ICloneable
+    {
+        ILogger GetLogger<T>();
+
+        ILogger GetLogger(Type type);
+
+        void LogMessage(ILogger logger, LogEventLevel level, string message);
+
+        ILogContext SetLevel(LogEventLevel level);
```

Comment:
`SetLevel()` seems non-native for C#. Using property would be better, but we need think about providing "one-line" configuration (Builder? and in future?).

Reply:
I had an opportunity to work on it and now we  can use it like
```csharp
Log.SetMinimumLevel(LogEventLevel.Debug)).WithHandler(new ConsoleLogHandler());
```